api_types.ts

```diff
@@ -1013,6 +1004,12 @@ export enum TableRole {
   GroupMembers = 'groupMembers',
 }
 
+export enum IncrementalPermissionCrawlType {
+  RecrawlAllUpdatedItems = 'recrawl_all_updated_items',
+  RecrawlOnlyNewItems = 'recrawl_only_new_items',
+  OnlyUseIncrementalPermissions = 'only_use_incremental_permissions',
```

Comment:
Aren't all incremental permission crawls "only using incremental permissions"?  I'm confused  If I understand the above comment correctly, should this be named something closer to "OnlyUseFullCrawling"?

Reply:
This enum is to control whether the non-incremental `executeGetPermissions` is invoked on sync results as they trickle into the ingestion processing (i.e., before we see the `completion` of the sync). I intend this particular option to indicate "never call executeGetPermissions, even if I'm returning a new row that doesn't yet exist in snowflake". So that's the opposite of "OnlyUseFullCrawling", so clearly I need to improve the naming here :)